packages/tools/devtools/devtools-view/src/components/AudienceHistoryTable.tsx

```diff
@@ -114,18 +112,14 @@ export function AudienceHistoryTable(props: AudienceHistoryTableProps): React.Re
 							themeInfo.name === ThemeOption.HighContrast
 								? style.highContrast
 								: item.changeKind === "joined"
-								? style.joined
-								: style.left
+									? style.joined
+									: style.left
 						}
 					>
 						<TableCell>
 							<LabelCellLayout
 								icon={
-									item.changeKind === "joined" ? (
-										<ArrowJoinRegular />
-									) : (
-										<ArrowExitRegular />
-									)
+									item.changeKind === "joined" ? <ArrowJoinRegular /> : <ArrowExitRegular />
```

Comment:
If this compiles, I love it. I'm not sure if the parents were actually required, when returning React components when using ternaries.